template.tpl

```diff
@@ -110,6 +118,10 @@ const setMilestones = () => {
   wv.getFID(process);
   wv.getCLS(process);
   wv.getLCP(process);
+  if (data.captureOtherWebVitals) {
+    wv.getTTFB(process);
+    wv.getFID(process);
```

Comment:
```suggestion
    wv.getFCP(process);
```
Looks like a typo after copy-paste getFID() method.

Reply:
Yeah, totally right. Will correct this!

Reply:
Pushed a fix